### PR TITLE
BUGFIX: make it possible for operators to reregister

### PR DESCRIPTION
**Pull request by ChaoticWalrus**
also add a minimal regression test, which simply checks that re-registering is possible!

### CODE REVIEW

File: test/unit/BLSRegistryCoordinatorWithIndicesUnit.t.sol [hunk 1]
```diff
@@ -579,6 +579,23 @@ contract BLSRegistryCoordinatorWithIndicesUnit is MockAVSDeployer {
...
+        bytes memory quorumNumbers = new bytes(1);
+        quorumNumbers[0] = bytes1(defaultQuorumNumber);
+
+        cheats.startPrank(defaultOperator);
+        
+        cheats.roll(registrationBlockNumber);
+        
+        registryCoordinator.registerOperatorWithCoordinator(quorumNumbers, defaultPubKey, defaultSocket);
+
+        uint256 quorumBitmap = BitmapUtils.orderedBytesArrayToBitmap(quorumNumbers);
```
Comment (gpsanant) [inline test/unit/BLSRegistryCoordinatorWithIndicesUnit.t.sol:597]: did you want to make an assertion about this

Comment (ChaoticWalrus) [reply test/unit/BLSRegistryCoordinatorWithIndicesUnit.t.sol:597]: ahh yeah good call, one sec I'll add one

Comment (ChaoticWalrus) [reply test/unit/BLSRegistryCoordinatorWithIndicesUnit.t.sol:603]: @gpsanant check this commit https://github.com/Layr-Labs/eigenlayer-middleware/pull/20/commits/aa1782de7e7b167706e3d1d7adfc82f3e7c5bbc9

**Review by gpsanant — APPROVED** [review]
lgtm

**Review by 0x0aa0 — APPROVED** [review]

**Pull request merged**